File: lab_05/PY/MainProgram.py

from Matrix2 import Matrix2
from Matrix3 import Matrix3

A = Matrix3()
B = Matrix2()

A.EnterMatrix()
B.EnterMatrix()

print(f"sum(A) = {A.Sum()}")
print(f"det(A) = {A.Det()}")
print(f"det(B) = {B.Det()}")

print(f"sum(A) + det(A) + det(B) = {A.Sum() + A.Det() + B.Det()}")

File: lab_05/CPP/MainProgram.cpp
#include "Matrix2.hpp"
#include "Matrix3.hpp"
#include <cstdio>


int main(int argc, char **argv)
{
    Matrix3 A;
    Matrix2 B;

    A.EnterMatrix();
    B.EnterMatrix();

    printf("sum(A) = %d\n", A.Sum());
    printf("det(A) = %d\n", A.Det());
    printf("det(B) = %d\n", B.Det());

    printf("sum(A) + det(A) + det(B) = %d\n", A.Sum() + A.Det() + B.Det());

    return 0;
}